TOC.md

```diff
@@ -73,6 +73,7 @@
   + Monitor
     - [Overview](/tidb-monitoring-framework.md)
     - [Monitor a TiDB Cluster](/monitor-a-tidb-cluster.md)
+    - [Export Grafana Snapshots](/exporting-grafana-snapshots.md)
```

Comment:
I add the link here. Please help confirm it. @kennytm @breeswish

Reply:
LGTM (same for the other two cherry-picks)